pkgs/development/python-modules/gocardless-pro/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, requests
+, six
+, pytestCheckHook
+, responses
+, nose
+,
+}:
+buildPythonPackage rec {
```

```suggestion
}:

buildPythonPackage rec {
```
This is a common pitfall when switching from alejandra to nixpkgs-fmt.